### PR TITLE
Migrate terra status

### LINKED ISSUE #66: Migrate terra-status into repo
https://github.com/cerner/terra-status

### PR DESCRIPTION
**Pull request by bjankord**
### Summary
Migrating terra-status component into monorepo.

Resolves #66

### CODE REVIEW

File: packages/terra-status/src/status.scss [hunk 1]
```diff
@@ -0,0 +1,6 @@
+@import '~terra-mixins';
```
Comment (bjankord) [inline packages/terra-status/src/status.scss]: This file needs to be checked in as `Status.scss`

File: packages/terra-status/LICENSE [hunk 2]
```diff
@@ -0,0 +1,202 @@
+
```
Comment (ryanthemanuel) [inline packages/terra-status/LICENSE]: At some point we need to add this (as well as NOTICE) to the existing projects (i.e. terra-button, etc.)  Should we just use this PR for that or create a new one?

Comment (bjankord) [inline packages/terra-status/LICENSE]: Good point. I'd like to do that work as a separate PR. I can reach out to @aganapuram and she if she's interested in taking that task on. I'll get a issue logged for the work as well.

File: packages/terra-status/docs/README.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+  - `npm install terra-status`
+  - `yarn install terra-status`
+
+## Usage
+
+```jsx
+import React from 'react';
+import Status from 'terra-status';
+
+<Status color="red">{simpleText}</Status>
```
Comment (Matt-Butler) [inline packages/terra-status/docs/README.md]: nit. replace {simpleText} with string

File: packages/terra-status/src/Status.jsx [hunk 4]
```diff
@@ -0,0 +1,30 @@
+import React, { PropTypes } from 'react';
+import classNames from 'classnames';
+import '../src/Status.scss';
```
Comment (Matt-Butler) [inline packages/terra-status/src/Status.jsx]: `'../src/Status.scss'` => `./Status.scss`

File: packages/terra-status/tests/Status.test.jsx [hunk 5]
```diff
@@ -0,0 +1,53 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import React from 'react';
+import Arrange from 'terra-arrange';
+import Status from '../src/Status';
+
+const image = <img src="http" alt="panda" />;
```
Comment (Matt-Butler) [inline packages/terra-status/tests/Status.test.jsx]: Do some of these src attributes need to be updated?

**Comment by amf9t2** [comment]
+1

File: packages/terra-status/src/status.scss (hunk 1 above)
Comment (bjankord) [inline packages/terra-status/src/status.scss]: Updated.

File: packages/terra-status/docs/README.md (hunk 3 above)
Comment (bjankord) [inline packages/terra-status/docs/README.md]: [Updated](https://github.com/cerner/terra-ui/pull/81/commits/b3efb70352a1e943a930dc16e42b21a9444031d7)

File: packages/terra-status/src/Status.jsx (hunk 4 above)
Comment (bjankord) [inline packages/terra-status/src/Status.jsx]: Updated:
* https://github.com/cerner/terra-ui/pull/81/commits/c5e1d689b96a77a18a8a1112231351a9d8922071
* https://github.com/cerner/terra-ui/pull/81/commits/32a42553e4ef76b76ce12053c1b2bda5cf154467

File: packages/terra-status/tests/Status.test.jsx (hunk 5 above)
Comment (bjankord) [inline packages/terra-status/tests/Status.test.jsx]: Hmm, looks like it, I'll take a look at this.

Comment (bjankord) [inline packages/terra-status/tests/Status.test.jsx]: Update `img src` across package: https://github.com/cerner/terra-ui/pull/81/commits/38e3350c04719132b9567c4fed2302d912bb088e

**Pull request merged**